userguide/jails.rst

```diff
@@ -1276,45 +1217,41 @@ is displayed:
 
 
 Use the arrow keys to select an option and press :kbd:`spacebar`
-to toggle the value. When all the values are as desired, press
-:kbd:`Enter`.  The port will begin to compile and install.
+to toggle the value. Press :kbd:`Enter` when satisfied with the jail
+options. The port begins to compile and install.
 
-.. note:: The configuration screen will not be shown again, even
-   if the build is stopped and restarted. It can be redisplayed
-   by typing :command:`make config`.  Change the settings, then
-   rebuild with :command:`make clean install clean`.
+.. note:: After options have been set, the configuration screen is
+   normally not shown again. Use :command:`make config` to display the
+   screen and change options before rebuilding the port with
+   :command:`make clean` install clean.
```

Comment:
The final backtick is after the final "clean":
:command:`make clean install clean`.